### PR TITLE
[Merged by Bors] - doc(Analysis): document some entries

### PR DESCRIPTION
**Pull request by grunweg**
---

See individual commit messages for details.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/BoxIntegral/Partition/Basic.lean [hunk 1]
```diff
@@ -53,8 +53,11 @@ variable {ι : Type*}
 /-- A prepartition of `I : BoxIntegral.Box ι` is a finite set of pairwise disjoint subboxes of
 `I`. -/
 structure Prepartition (I : Box ι) where
+  /-- The underlying set of boxes -/
   boxes : Finset (Box ι)
+  /-- Each box is a sub-box of `I` -/
   le_of_mem' : ∀ J ∈ boxes, J ≤ I
+  /-- The boxes in a prepartition are pairwise disjoing -/
```
Comment (urkud) [inline Mathlib/Analysis/BoxIntegral/Partition/Basic.lean:60]:
```suggestion
  /-- The boxes in a prepartition are pairwise disjoint. -/
```

File: Mathlib/Analysis/BoxIntegral/Partition/Tagged.lean [hunk 2]
```diff
@@ -40,7 +40,9 @@ variable {ι : Type*}
 prepartition. For simplicity we require that `tag` is defined for all boxes in `ι → ℝ` but
 we will use only the values of `tag` on the boxes of the partition. -/
 structure TaggedPrepartition (I : Box ι) extends Prepartition I where
+  /-- Choice of tagged point of each box in this prepartition -/
```
Comment (urkud) [inline Mathlib/Analysis/BoxIntegral/Partition/Tagged.lean:43]: Please mention that we extend it to a total function on all boxes.

**Comment by urkud** [comment]
Otherwise LGTM. Thanks!
bors d+

**Comment by grunweg** [comment]
Thank you for the review and the good suggestions.
bors r+

**Pull request closed without merging**